meson/build-cli.sh

```diff
@@ -1,7 +0,0 @@
-#!/bin/bash
-
-exec_name=writeas
-
-echo "Building $exec_name CLI..."
-gb build github.com/writeas/writeas-cli/cmd/writeas &&
-echo "Success."
```

Comment:
I believe this script is still useful... Any reason we should get rid of it?

Reply:
Added it back